Makefile

```diff
@@ -18,10 +18,28 @@ all: iso
 debug ?= none
 net ?= none
 
+## Should we use the limine bootloader?
+use_limine ?= NO
```

Comment:
I think it's more typical to set up a standard argument like the ones above, and give it a default value. For example:
```make
boot ?= grub
```
And then below, check for invalid values of `boot`, just like we do for `debug` or `net`.

Reply:
Correspondingly, we should rename the `boot` target to `usb` since that's what it actually is for.